### PR TITLE
Handle new title in Special:NewFiles page

### PR DESCRIPTION
**Pull request by ukaka**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 1 major
* MINOR 1 minor
* INFO 1 info

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Add the missing @deprecated Javadoc tag.](https://github.com/Wikia/selenium-tests/blob/03b8febce4dde7903770d9befa74b2d95bbf5673/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java#L158) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/Wikia/selenium-tests/blob/03b8febce4dde7903770d9befa74b2d95bbf5673/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00105)
* INFO [Do not forget to remove this deprecated code someday.](https://github.com/Wikia/selenium-tests/blob/03b8febce4dde7903770d9befa74b2d95bbf5673/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java#L158) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java [hunk 1]
```diff
@@ -211,4 +213,8 @@ public String getImageKey(int itemNumber) {
     return imageKey;
   }
 
+  public String getNewFilesSpecialPageTitle() {
+	return NEW_FILES_SPECIAL_PAGE_TITLE;
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java]: Wrong identation

File: src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java [hunk 2]
```diff
@@ -9,6 +9,7 @@
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.signup.SignUpPageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialNewFilesPageObject;
+import static com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialPageObject.NEW_FILES_SPECIAL_PAGE_TITLE;
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java]: Compilation error

File: src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java [hunk 3]
```diff
@@ -47,7 +48,7 @@ public void FBButton_003_LoginButton_Visible() {
   public void FBButton_004_ForcedLoginButton_Visible() {
     WikiBasePageObject base = new WikiBasePageObject(driver);
     SpecialNewFilesPageObject specialPage = base.openSpecialNewFiles(wikiURL);
-    specialPage.verifySpecialPage();
+    specialPage.verifyPageHeader(NEW_FILES_SPECIAL_PAGE_TITLE);
```
Comment (bulaszenko) [inline src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java]: It should be specialPage.verifyPageHeader(specialPage.getNewFilesSpecialPageTitle());

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 1 major
* MINOR 1 minor
* INFO 1 info

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Add the missing @deprecated Javadoc tag.](https://github.com/Wikia/selenium-tests/blob/744d3082a57b3a86de3c56f3eb0ad648d646975d/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java#L158) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
* INFO [Do not forget to remove this deprecated code someday.](https://github.com/Wikia/selenium-tests/blob/744d3082a57b3a86de3c56f3eb0ad648d646975d/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java#L158) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/Wikia/selenium-tests/blob/744d3082a57b3a86de3c56f3eb0ad648d646975d/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00105)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java [hunk 4]
```diff
@@ -211,4 +213,8 @@ public String getImageKey(int itemNumber) {
     return imageKey;
   }
 
+  public String getNewFilesSpecialPageTitle() {
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java]: this method is redundant, you can call field directly if you change it to public

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java [hunk 5]
```diff
@@ -211,4 +213,8 @@ public String getImageKey(int itemNumber) {
...
+    return NEW_FILES_SPECIAL_PAGE_TITLE;
+  }
+
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java]: remove extra line

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java [hunk 6]
```diff
@@ -20,6 +20,8 @@
 
 public class SpecialNewFilesPageObject extends SpecialPageObject {
 
+  private static final String NEW_FILES_SPECIAL_PAGE_TITLE = "Images";
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java]: how does `NEW_FILES_SPECIAL_PAGE_TITLE` has sth to do with `Images`?

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java (hunk 4 above)
Comment (drets) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPageObject.java]: :+1:

**Comment by Archbold** [comment]
LGTM Tests pass after merging branch locally and running tests.
Will merge PR to master

**Comment by tomasznapieralski** [comment]
@Archbold did you pushed your changes? i don't see changes i was asking for or replies to my comments

**Comment by Archbold** [comment]
Yes I pushed. Your comments weren't visible when I pushed.
To answer your question, the title of the Special:NewFiles page is now "Images" See attached ScreenCapture 08-Jan-16 01.PNG
screencapture 08-jan-16 01

**Pull request merged**